File: Python/tests/test_prl_ewa.py
import pytest

from hbayesdm.models import prl_ewa


def test_prl_ewa():
    _ = prl_ewa(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_prl_ewa.R
context("Test prl_ewa")
library(hBayesDM)

test_that("Test prl_ewa", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(prl_ewa(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
